2019.05.13 15:31:45 LOG5[ui]: stunnel 5.51 on x86_64-pc-linux-gnu platform
2019.05.13 15:31:45 LOG5[ui]: Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
2019.05.13 15:31:45 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
2019.05.13 15:31:45 LOG5[ui]: Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
2019.05.13 15:31:45 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.13 15:31:45 LOG5[ui]: FIPS mode disabled
2019.05.13 15:31:45 LOG4[ui]: Insecure file permissions on stunnel/stunnel.pem
2019.05.13 15:31:45 LOG5[ui]: Configuration successful
2019.05.13 15:31:45 LOG5[ui]: Binding service [https] to :::8443: Address already in use (98)
2019.05.13 15:34:01 LOG5[0]: Service [https] accepted connection from 127.0.0.1:35350
2019.05.13 15:34:01 LOG3[0]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:34:01 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:34:01 LOG5[1]: Service [https] accepted connection from 127.0.0.1:35354
2019.05.13 15:34:01 LOG5[2]: Service [https] accepted connection from 127.0.0.1:35358
2019.05.13 15:34:01 LOG3[1]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:34:01 LOG5[3]: Service [https] accepted connection from 127.0.0.1:35362
2019.05.13 15:34:01 LOG3[2]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:34:01 LOG5[2]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:34:01 LOG3[3]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:34:01 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:34:01 LOG5[3]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:34:01 LOG5[4]: Service [https] accepted connection from 127.0.0.1:35366
2019.05.13 15:34:01 LOG3[4]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:34:01 LOG5[4]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:34:01 LOG5[5]: Service [https] accepted connection from 127.0.0.1:35370
2019.05.13 15:34:01 LOG3[5]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:34:01 LOG5[5]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:34:54 LOG3[ui]: Received SIGINT; terminating
2019.05.13 15:35:21 LOG5[ui]: stunnel 5.51 on x86_64-pc-linux-gnu platform
2019.05.13 15:35:21 LOG5[ui]: Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
2019.05.13 15:35:21 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
2019.05.13 15:35:21 LOG5[ui]: Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
2019.05.13 15:35:21 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.13 15:35:21 LOG5[ui]: FIPS mode disabled
2019.05.13 15:35:21 LOG4[ui]: Insecure file permissions on stunnel/stunnel.pem
2019.05.13 15:35:21 LOG5[ui]: Configuration successful
2019.05.13 15:35:21 LOG5[ui]: Binding service [https] to :::8443: Address already in use (98)
2019.05.13 15:35:25 LOG3[ui]: Received SIGINT; terminating
2019.05.13 15:35:41 LOG5[ui]: stunnel 5.51 on x86_64-pc-linux-gnu platform
2019.05.13 15:35:41 LOG5[ui]: Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
2019.05.13 15:35:41 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
2019.05.13 15:35:41 LOG5[ui]: Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
2019.05.13 15:35:41 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.13 15:35:41 LOG5[ui]: FIPS mode disabled
2019.05.13 15:35:41 LOG4[ui]: Insecure file permissions on stunnel/stunnel.pem
2019.05.13 15:35:41 LOG5[ui]: Configuration successful
2019.05.13 15:35:41 LOG5[ui]: Binding service [https] to :::8443: Address already in use (98)
2019.05.13 15:35:46 LOG5[0]: Service [https] accepted connection from 127.0.0.1:36228
2019.05.13 15:35:46 LOG3[0]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:35:46 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:35:46 LOG5[2]: Service [https] accepted connection from 127.0.0.1:36236
2019.05.13 15:35:46 LOG3[2]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:35:46 LOG5[2]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:35:46 LOG5[1]: Service [https] accepted connection from 127.0.0.1:36232
2019.05.13 15:35:46 LOG3[1]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:35:46 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:35:46 LOG5[3]: Service [https] accepted connection from 127.0.0.1:36240
2019.05.13 15:35:46 LOG3[3]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:35:46 LOG5[3]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:24 LOG5[4]: Service [https] accepted connection from 127.0.0.1:37150
2019.05.13 15:37:24 LOG5[5]: Service [https] accepted connection from 127.0.0.1:37152
2019.05.13 15:37:24 LOG3[4]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:24 LOG3[5]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:24 LOG5[4]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:24 LOG5[5]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:24 LOG5[6]: Service [https] accepted connection from 127.0.0.1:37154
2019.05.13 15:37:24 LOG3[6]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:24 LOG5[6]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:24 LOG5[7]: Service [https] accepted connection from 127.0.0.1:37156
2019.05.13 15:37:24 LOG3[7]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:24 LOG5[7]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:24 LOG5[8]: Service [https] accepted connection from 127.0.0.1:37158
2019.05.13 15:37:24 LOG3[8]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:24 LOG5[8]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:24 LOG5[9]: Service [https] accepted connection from 127.0.0.1:37164
2019.05.13 15:37:24 LOG3[9]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:24 LOG5[9]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:55 LOG5[10]: Service [https] accepted connection from 127.0.0.1:37426
2019.05.13 15:37:55 LOG3[10]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:55 LOG5[10]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:55 LOG5[12]: Service [https] accepted connection from 127.0.0.1:37434
2019.05.13 15:37:55 LOG3[12]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:55 LOG5[12]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:37:55 LOG5[11]: Service [https] accepted connection from 127.0.0.1:37430
2019.05.13 15:37:55 LOG3[11]: SSL_accept: ssl/statem/statem_lib.c:110: error:141FC044:SSL routines:tls_setup_handshake:internal error
2019.05.13 15:37:55 LOG5[11]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:39:34 LOG3[ui]: Received SIGINT; terminating
2019.05.13 15:40:13 LOG5[ui]: stunnel 5.51 on x86_64-pc-linux-gnu platform
2019.05.13 15:40:13 LOG5[ui]: Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
2019.05.13 15:40:13 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
2019.05.13 15:40:13 LOG5[ui]: Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
2019.05.13 15:40:13 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.13 15:40:13 LOG5[ui]: FIPS mode disabled
2019.05.13 15:40:13 LOG4[ui]: Insecure file permissions on stunnel/stunnel.pem
2019.05.13 15:40:13 LOG5[ui]: Configuration successful
2019.05.13 15:40:13 LOG5[ui]: Binding service [https] to :::8443: Address already in use (98)
2019.05.13 15:40:25 LOG5[0]: Service [https] accepted connection from 127.0.0.1:38720
2019.05.13 15:40:25 LOG3[0]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:25 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:25 LOG5[1]: Service [https] accepted connection from 127.0.0.1:38724
2019.05.13 15:40:25 LOG3[1]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:25 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:26 LOG5[3]: Service [https] accepted connection from 127.0.0.1:38740
2019.05.13 15:40:26 LOG5[2]: Service [https] accepted connection from 127.0.0.1:38736
2019.05.13 15:40:26 LOG3[3]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:26 LOG3[2]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:26 LOG5[2]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:26 LOG5[3]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:34 LOG5[4]: Service [https] accepted connection from 127.0.0.1:38820
2019.05.13 15:40:34 LOG5[5]: Service [https] accepted connection from 127.0.0.1:38824
2019.05.13 15:40:34 LOG3[5]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:34 LOG5[5]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:34 LOG3[4]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:34 LOG5[4]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:57 LOG5[6]: Service [https] accepted connection from 127.0.0.1:39004
2019.05.13 15:40:57 LOG5[7]: Service [https] accepted connection from 127.0.0.1:39008
2019.05.13 15:40:57 LOG3[6]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:57 LOG5[6]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:57 LOG3[7]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:57 LOG5[7]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:57 LOG5[8]: Service [https] accepted connection from 127.0.0.1:39012
2019.05.13 15:40:57 LOG3[8]: SSL_accept: ssl/record/rec_layer_s3.c:1536: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
2019.05.13 15:40:57 LOG5[8]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.13 15:40:57 LOG5[9]: Service [https] accepted connection from 127.0.0.1:39016
2019.05.13 15:40:57 LOG3[9]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.13 15:40:57 LOG5[9]: s_connect: connected 127.0.0.1:8888
2019.05.13 15:40:57 LOG5[9]: Service [https] connected remote server from 127.0.0.1:56170
2019.05.13 15:49:14 LOG3[ui]: Received SIGINT; terminating
2019.05.14 09:34:39 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 09:34:39 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 09:34:39 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 09:34:39 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 09:34:39 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 09:34:39 LOG5[ui]: FIPS mode disabled
2019.05.14 09:34:39 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 09:34:39 LOG5[ui]: Configuration successful
2019.05.14 09:35:18 LOG5[ui]: Terminated
2019.05.14 09:35:21 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 09:35:21 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 09:35:21 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 09:35:21 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 09:35:21 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 09:35:21 LOG5[ui]: FIPS mode disabled
2019.05.14 09:35:21 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 09:35:21 LOG5[ui]: Configuration successful
2019.05.14 10:36:52 LOG5[0]: Service [https] accepted connection from 91.107.64.185:46570
2019.05.14 10:36:52 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.14 10:36:52 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53172
2019.05.14 10:36:52 LOG5[0]: Connection closed: 266 byte(s) sent to TLS, 177 byte(s) sent to socket
2019.05.14 11:37:34 LOG5[1]: Service [https] accepted connection from 162.243.144.166:58872
2019.05.14 11:37:34 LOG3[1]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 11:37:34 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.14 11:37:34 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53300
2019.05.14 11:37:44 LOG5[1]: Connection closed: 266 byte(s) sent to TLS, 111 byte(s) sent to socket
2019.05.14 12:13:40 LOG5[2]: Service [https] accepted connection from 92.118.37.91:35096
2019.05.14 12:13:40 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.14 12:13:40 LOG5[2]: Service [https] connected remote server from 127.0.0.1:53376
2019.05.14 12:13:40 LOG5[2]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 12:19:42 LOG5[3]: Service [https] accepted connection from 92.118.37.91:39024
2019.05.14 12:19:43 LOG3[3]: SSL_accept: Peer suddenly disconnected
2019.05.14 12:19:43 LOG5[3]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 12:21:40 LOG5[4]: Service [https] accepted connection from 92.118.37.91:41540
2019.05.14 12:21:46 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.14 12:21:46 LOG5[4]: Service [https] connected remote server from 127.0.0.1:53394
2019.05.14 12:21:48 LOG5[4]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 12:27:39 LOG5[5]: Service [https] accepted connection from 176.58.124.134:38660
2019.05.14 12:27:39 LOG3[5]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.14 12:27:39 LOG5[5]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:07:36 LOG5[6]: Service [https] accepted connection from 61.219.11.153:61643
2019.05.14 14:07:38 LOG3[6]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.14 14:07:38 LOG5[6]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:51:31 LOG5[ui]: Terminated
2019.05.14 14:51:35 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:51:35 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:51:35 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:51:35 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:51:35 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:51:35 LOG5[ui]: FIPS mode disabled
2019.05.14 14:51:35 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:51:35 LOG5[ui]: Configuration successful
2019.05.14 14:53:16 LOG5[ui]: Terminated
2019.05.14 14:53:19 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:53:19 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:53:19 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:53:19 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:53:19 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:53:19 LOG5[ui]: FIPS mode disabled
2019.05.14 14:53:19 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:53:19 LOG5[ui]: Configuration successful
2019.05.14 14:53:55 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17491
2019.05.14 14:54:00 LOG3[0]: SSL_accept: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2019.05.14 14:54:00 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:54:53 LOG5[ui]: Terminated
2019.05.14 14:54:55 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:54:55 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:54:55 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:54:55 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:54:55 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:54:55 LOG5[ui]: FIPS mode disabled
2019.05.14 14:54:55 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:54:55 LOG5[ui]: Configuration successful
2019.05.14 14:55:07 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17529
2019.05.14 14:55:10 LOG3[0]: SSL_accept: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2019.05.14 14:55:10 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:55:18 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17537
2019.05.14 14:55:19 LOG3[1]: SSL_accept: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2019.05.14 14:55:19 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:55:27 LOG5[2]: Service [https] accepted connection from 183.192.90.7:17545
2019.05.14 14:55:28 LOG5[3]: Service [https] accepted connection from 183.192.90.7:17546
2019.05.14 14:55:28 LOG3[2]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 14:55:28 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.14 14:55:28 LOG5[2]: Service [https] connected remote server from 127.0.0.1:53972
2019.05.14 14:55:28 LOG3[3]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 14:55:28 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.14 14:55:28 LOG5[3]: Service [https] connected remote server from 127.0.0.1:53976
2019.05.14 14:55:29 LOG5[4]: Service [https] accepted connection from 183.192.90.7:17548
2019.05.14 14:55:32 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.14 14:55:32 LOG5[4]: Service [https] connected remote server from 127.0.0.1:53980
2019.05.14 14:57:42 LOG3[4]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.14 14:57:42 LOG5[4]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:57:42 LOG3[3]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.14 14:57:42 LOG5[3]: Connection closed: 266 byte(s) sent to TLS, 306 byte(s) sent to socket
2019.05.14 14:57:42 LOG3[2]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.14 14:57:42 LOG5[2]: Connection closed: 748 byte(s) sent to TLS, 1273 byte(s) sent to socket
2019.05.14 14:57:42 LOG5[ui]: Terminated
2019.05.14 14:57:45 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:57:45 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:57:45 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:57:45 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:57:45 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:57:45 LOG5[ui]: FIPS mode disabled
2019.05.14 14:57:45 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:57:45 LOG5[ui]: Configuration successful
2019.05.14 15:11:28 LOG5[0]: Service [https] accepted connection from 183.192.90.7:18471
2019.05.14 15:11:29 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:11:29 LOG5[0]: Service [https] connected remote server from 127.0.0.1:54600
2019.05.14 15:11:32 LOG5[0]: Connection closed: 266 byte(s) sent to TLS, 385 byte(s) sent to socket
2019.05.14 15:12:28 LOG5[1]: Service [https] accepted connection from 61.151.178.177:42841
2019.05.14 15:12:28 LOG3[1]: SSL_accept: Peer suddenly disconnected
2019.05.14 15:12:28 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 15:18:11 LOG5[2]: Service [https] accepted connection from 183.192.90.7:18882
2019.05.14 15:18:11 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:18:11 LOG5[2]: Service [https] connected remote server from 127.0.0.1:54780
2019.05.14 15:18:24 LOG5[2]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 15:19:48 LOG5[3]: Service [https] accepted connection from 183.192.172.11:19498
2019.05.14 15:19:50 LOG3[3]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 15:19:50 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:19:50 LOG5[3]: Service [https] connected remote server from 127.0.0.1:54828
2019.05.14 15:19:51 LOG5[3]: Connection closed: 266 byte(s) sent to TLS, 152 byte(s) sent to socket
2019.05.14 15:20:24 LOG5[4]: Service [https] accepted connection from 183.192.172.11:19500
2019.05.14 15:20:25 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:20:25 LOG5[4]: Service [https] connected remote server from 127.0.0.1:54842
2019.05.14 15:20:25 LOG5[4]: Connection closed: 266 byte(s) sent to TLS, 152 byte(s) sent to socket
2019.05.14 17:15:30 LOG5[5]: Service [https] accepted connection from 122.228.19.79:29614
2019.05.14 17:15:30 LOG3[5]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 17:15:30 LOG5[5]: s_connect: connected 127.0.0.1:8888
2019.05.14 17:15:30 LOG5[5]: Service [https] connected remote server from 127.0.0.1:58040
2019.05.14 17:15:31 LOG5[5]: Connection closed: 266 byte(s) sent to TLS, 177 byte(s) sent to socket
2019.05.14 20:26:10 LOG5[6]: Service [https] accepted connection from 107.170.200.205:59360
2019.05.14 20:26:12 LOG3[6]: SSL_accept: 1408A0C1: error:1408A0C1:SSL routines:ssl3_get_client_hello:no shared cipher
2019.05.14 20:26:12 LOG5[6]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 20:57:50 LOG5[7]: Service [https] accepted connection from 195.154.61.206:43565
2019.05.14 20:57:51 LOG3[7]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 20:57:51 LOG5[7]: s_connect: connected 127.0.0.1:8888
2019.05.14 20:57:51 LOG5[7]: Service [https] connected remote server from 127.0.0.1:35616
2019.05.14 20:58:04 LOG5[7]: Connection closed: 266 byte(s) sent to TLS, 238 byte(s) sent to socket
2019.05.14 21:18:32 LOG5[8]: Service [https] accepted connection from 162.243.151.182:52342
2019.05.14 21:18:32 LOG5[8]: s_connect: connected 127.0.0.1:8888
2019.05.14 21:18:32 LOG5[8]: Service [https] connected remote server from 127.0.0.1:36158
2019.05.14 21:18:32 LOG5[8]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 23:45:46 LOG5[9]: Service [https] accepted connection from 196.52.43.90:56319
2019.05.14 23:45:48 LOG3[9]: SSL_accept: Peer suddenly disconnected
2019.05.14 23:45:48 LOG5[9]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 00:53:16 LOG5[10]: Service [https] accepted connection from 184.105.139.68:51774
2019.05.15 00:53:19 LOG5[10]: s_connect: connected 127.0.0.1:8888
2019.05.15 00:53:19 LOG5[10]: Service [https] connected remote server from 127.0.0.1:41780
2019.05.15 00:53:20 LOG5[10]: Connection closed: 266 byte(s) sent to TLS, 40 byte(s) sent to socket
2019.05.15 00:54:54 LOG5[11]: Service [https] accepted connection from 184.105.139.68:9700
2019.05.15 00:54:54 LOG3[11]: SSL_accept: 1408A10B: error:1408A10B:SSL routines:ssl3_get_client_hello:wrong version number
2019.05.15 00:54:54 LOG5[11]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 00:55:20 LOG5[12]: Service [https] accepted connection from 184.105.139.68:6962
2019.05.15 00:55:20 LOG3[12]: SSL_accept: 1408A0C1: error:1408A0C1:SSL routines:ssl3_get_client_hello:no shared cipher
2019.05.15 00:55:20 LOG5[12]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 00:55:44 LOG5[13]: Service [https] accepted connection from 184.105.139.68:4214
2019.05.15 00:55:45 LOG3[13]: SSL_accept: 1408A0C1: error:1408A0C1:SSL routines:ssl3_get_client_hello:no shared cipher
2019.05.15 00:55:45 LOG5[13]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 01:19:00 LOG5[14]: Service [https] accepted connection from 104.248.85.56:39922
2019.05.15 01:19:00 LOG3[14]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 01:19:00 LOG5[14]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 01:32:40 LOG5[15]: Service [https] accepted connection from 71.6.232.7:46034
2019.05.15 01:32:41 LOG3[15]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.15 01:32:41 LOG5[15]: s_connect: connected 127.0.0.1:8888
2019.05.15 01:32:41 LOG5[15]: Service [https] connected remote server from 127.0.0.1:42808
2019.05.15 01:32:48 LOG5[15]: Connection closed: 266 byte(s) sent to TLS, 205 byte(s) sent to socket
2019.05.15 03:26:05 LOG5[16]: Service [https] accepted connection from 131.153.31.59:51751
2019.05.15 03:26:05 LOG5[16]: s_connect: connected 127.0.0.1:8888
2019.05.15 03:26:05 LOG5[16]: Service [https] connected remote server from 127.0.0.1:45756
2019.05.15 03:26:05 LOG5[16]: Connection closed: 266 byte(s) sent to TLS, 78 byte(s) sent to socket
2019.05.15 05:10:59 LOG5[17]: Service [https] accepted connection from 198.108.66.240:5572
2019.05.15 05:10:59 LOG3[17]: SSL_accept: 1408A10B: error:1408A10B:SSL routines:ssl3_get_client_hello:wrong version number
2019.05.15 05:10:59 LOG5[17]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 06:36:36 LOG5[18]: Service [https] accepted connection from 176.58.124.134:57266
2019.05.15 06:36:36 LOG3[18]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 06:36:36 LOG5[18]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 07:02:25 LOG5[19]: Service [https] accepted connection from 164.52.24.162:49001
2019.05.15 07:02:25 LOG3[19]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.15 07:02:25 LOG5[19]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 07:16:05 LOG5[20]: Service [https] accepted connection from 139.162.113.204:39724
2019.05.15 07:16:05 LOG5[20]: s_connect: connected 127.0.0.1:8888
2019.05.15 07:16:05 LOG5[20]: Service [https] connected remote server from 127.0.0.1:51818
2019.05.15 07:16:06 LOG5[20]: Connection closed: 266 byte(s) sent to TLS, 122 byte(s) sent to socket
2019.05.15 09:04:04 LOG5[21]: Service [https] accepted connection from 178.128.194.144:44324
2019.05.15 09:04:05 LOG3[21]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.15 09:04:05 LOG5[21]: s_connect: connected 127.0.0.1:8888
2019.05.15 09:04:05 LOG5[21]: Service [https] connected remote server from 127.0.0.1:54648
2019.05.15 09:04:05 LOG5[21]: Connection closed: 266 byte(s) sent to TLS, 207 byte(s) sent to socket
2019.05.15 09:04:05 LOG5[22]: Service [https] accepted connection from 178.128.194.144:44612
2019.05.15 09:04:05 LOG3[22]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.15 09:04:05 LOG5[22]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 09:04:05 LOG5[23]: Service [https] accepted connection from 178.128.194.144:44756
2019.05.15 09:04:05 LOG3[23]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 09:04:05 LOG5[23]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 09:24:04 LOG5[24]: Service [https] accepted connection from 124.207.119.80:50436
2019.05.15 09:24:05 LOG5[24]: s_connect: connected 127.0.0.1:8888
2019.05.15 09:24:05 LOG5[24]: Service [https] connected remote server from 127.0.0.1:55168
2019.05.15 09:24:14 LOG5[24]: Connection closed: 266 byte(s) sent to TLS, 111 byte(s) sent to socket
2019.05.15 09:24:34 LOG5[25]: Service [https] accepted connection from 35.246.247.9:53824
2019.05.15 09:24:34 LOG3[25]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.15 09:24:34 LOG5[25]: s_connect: connected 127.0.0.1:8888
2019.05.15 09:24:34 LOG5[25]: Service [https] connected remote server from 127.0.0.1:55186
2019.05.15 09:24:34 LOG5[25]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 10:53:58 LOG5[26]: Service [https] accepted connection from 106.75.3.52:53780
2019.05.15 10:53:58 LOG3[26]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.15 10:53:58 LOG5[26]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 10:54:04 LOG5[27]: Service [https] accepted connection from 106.75.3.52:57382
2019.05.15 10:54:04 LOG3[27]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 10:54:04 LOG5[27]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:03:10 LOG5[28]: Service [https] accepted connection from 108.178.16.154:39392
2019.05.15 11:03:10 LOG3[28]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:03:10 LOG5[28]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:03:13 LOG5[29]: Service [https] accepted connection from 108.178.16.154:41488
2019.05.15 11:03:13 LOG3[29]: SSL_accept: 1408A0A0: error:1408A0A0:SSL routines:ssl3_get_client_hello:length too short
2019.05.15 11:03:13 LOG5[29]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:17:45 LOG3[30]: getpeerbyname (local_rfd): Transport endpoint is not connected (107)
2019.05.15 11:17:45 LOG5[30]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:17:45 LOG5[31]: Service [https] accepted connection from 92.119.160.11:22347
2019.05.15 11:17:45 LOG3[31]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:17:45 LOG5[31]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:17:45 LOG5[32]: Service [https] accepted connection from 92.119.160.11:22859
2019.05.15 11:17:45 LOG3[32]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:17:45 LOG5[32]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:45:56 LOG5[33]: Service [https] accepted connection from 185.10.68.172:45728
2019.05.15 11:45:56 LOG3[33]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:45:56 LOG5[33]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:45:56 LOG5[34]: Service [https] accepted connection from 185.10.68.172:45876
2019.05.15 11:45:56 LOG3[34]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:45:56 LOG5[34]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 16:38:23 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:38:23 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:38:23 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:38:23 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:38:23 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:38:23 LOG5[ui]: FIPS mode disabled
2019.05.16 16:38:23 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:38:23 LOG5[ui]: Configuration successful
2019.05.16 16:38:52 LOG5[ui]: Terminated
2019.05.16 16:38:56 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:38:56 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:38:56 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:38:56 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:38:56 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:38:56 LOG5[ui]: FIPS mode disabled
2019.05.16 16:38:56 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:38:56 LOG5[ui]: Configuration successful
2019.05.16 16:39:14 LOG5[0]: Service [https] accepted connection from 183.192.90.7:19337
2019.05.16 16:39:16 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:16 LOG5[0]: Service [https] connected remote server from 127.0.0.1:52822
2019.05.16 16:39:16 LOG5[1]: Service [https] accepted connection from 183.192.90.7:19335
2019.05.16 16:39:19 LOG3[1]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 16:39:19 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:19 LOG5[1]: Service [https] connected remote server from 127.0.0.1:52828
2019.05.16 16:39:32 LOG5[1]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 16:39:33 LOG5[3]: Service [https] accepted connection from 183.192.90.7:19354
2019.05.16 16:39:33 LOG5[4]: Service [https] accepted connection from 183.192.90.7:19356
2019.05.16 16:39:33 LOG5[5]: Service [https] accepted connection from 183.192.90.7:19352
2019.05.16 16:39:33 LOG5[2]: Service [https] accepted connection from 183.192.90.7:19351
2019.05.16 16:39:33 LOG5[6]: Service [https] accepted connection from 183.192.90.7:19353
2019.05.16 16:39:33 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:33 LOG5[3]: Service [https] connected remote server from 127.0.0.1:52836
2019.05.16 16:39:33 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:33 LOG5[4]: Service [https] connected remote server from 127.0.0.1:52838
2019.05.16 16:39:33 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:33 LOG5[2]: Service [https] connected remote server from 127.0.0.1:52840
2019.05.16 16:39:33 LOG5[5]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:33 LOG5[5]: Service [https] connected remote server from 127.0.0.1:52842
2019.05.16 16:39:33 LOG5[6]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:39:33 LOG5[6]: Service [https] connected remote server from 127.0.0.1:52844
2019.05.16 16:43:55 LOG5[7]: Service [https] accepted connection from 180.160.228.253:40322
2019.05.16 16:43:55 LOG5[8]: Service [https] accepted connection from 180.160.228.253:40334
2019.05.16 16:43:56 LOG5[7]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:43:56 LOG5[7]: Service [https] connected remote server from 127.0.0.1:52958
2019.05.16 16:44:59 LOG5[3]: Connection closed: 138023 byte(s) sent to TLS, 1014 byte(s) sent to socket
2019.05.16 16:44:59 LOG5[4]: Connection closed: 73027 byte(s) sent to TLS, 1003 byte(s) sent to socket
2019.05.16 16:44:59 LOG5[2]: Connection closed: 91352 byte(s) sent to TLS, 1003 byte(s) sent to socket
2019.05.16 16:44:59 LOG5[5]: Connection closed: 65722 byte(s) sent to TLS, 964 byte(s) sent to socket
2019.05.16 16:45:04 LOG5[ui]: Terminated
2019.05.16 16:45:07 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:45:07 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:45:07 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:45:07 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:45:07 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:45:07 LOG5[ui]: FIPS mode disabled
2019.05.16 16:45:07 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:45:07 LOG5[ui]: Configuration successful
2019.05.16 16:45:27 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17496
2019.05.16 16:45:27 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17497
2019.05.16 16:45:27 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:45:27 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53000
2019.05.16 16:45:28 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:45:28 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53004
2019.05.16 16:45:43 LOG5[1]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 16:46:26 LOG3[0]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.16 16:46:26 LOG5[0]: Connection closed: 683 byte(s) sent to TLS, 14968 byte(s) sent to socket
2019.05.16 16:46:51 LOG5[2]: Service [https] accepted connection from 183.192.90.7:17506
2019.05.16 16:46:52 LOG5[3]: Service [https] accepted connection from 183.192.90.7:17507
2019.05.16 16:46:52 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:46:52 LOG5[2]: Service [https] connected remote server from 127.0.0.1:53042
2019.05.16 16:46:52 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:46:52 LOG5[3]: Service [https] connected remote server from 127.0.0.1:53044
2019.05.16 16:47:05 LOG5[3]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 16:50:18 LOG5[ui]: Terminated
2019.05.16 16:50:21 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:50:21 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:50:21 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:50:21 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:50:21 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:50:21 LOG5[ui]: FIPS mode disabled
2019.05.16 16:50:21 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:50:21 LOG5[ui]: Configuration successful
2019.05.16 16:50:34 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17563
2019.05.16 16:50:35 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17564
2019.05.16 16:50:35 LOG3[1]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 16:50:35 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:50:35 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53150
2019.05.16 16:50:36 LOG3[0]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 16:50:36 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:50:36 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53154
2019.05.16 16:51:43 LOG5[0]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 16:51:43 LOG5[2]: Service [https] accepted connection from 183.192.90.7:17571
2019.05.16 16:51:44 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:51:44 LOG5[2]: Service [https] connected remote server from 127.0.0.1:53186
2019.05.16 16:52:47 LOG5[ui]: Terminated
2019.05.16 16:52:50 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:52:50 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:52:50 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:52:50 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:52:50 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:52:50 LOG5[ui]: FIPS mode disabled
2019.05.16 16:52:50 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:52:50 LOG5[ui]: Configuration successful
2019.05.16 16:53:06 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17581
2019.05.16 16:53:06 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17582
2019.05.16 16:53:06 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:53:06 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53222
2019.05.16 16:53:06 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:53:06 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53224
2019.05.16 16:54:31 LOG5[ui]: Terminated
2019.05.16 16:54:34 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:54:34 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:54:34 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:54:34 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:54:34 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:54:34 LOG5[ui]: FIPS mode disabled
2019.05.16 16:54:34 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:54:34 LOG5[ui]: Configuration successful
2019.05.16 16:55:49 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17785
2019.05.16 16:55:50 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17790
2019.05.16 16:55:54 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:55:54 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53300
2019.05.16 16:56:01 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:56:01 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53304
2019.05.16 16:56:23 LOG5[0]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 16:56:23 LOG5[1]: Connection closed: 0 byte(s) sent to TLS, 14337 byte(s) sent to socket
2019.05.16 16:57:59 LOG5[ui]: Terminated
2019.05.16 16:58:02 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 16:58:02 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 16:58:02 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 16:58:02 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 16:58:02 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 16:58:02 LOG5[ui]: FIPS mode disabled
2019.05.16 16:58:02 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 16:58:02 LOG5[ui]: Configuration successful
2019.05.16 16:58:20 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17829
2019.05.16 16:58:20 LOG3[0]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 16:58:20 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:58:20 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53370
2019.05.16 16:58:22 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17830
2019.05.16 16:58:22 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 16:58:22 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53372
2019.05.16 16:58:37 LOG5[1]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 17:00:14 LOG3[0]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.16 17:00:14 LOG5[0]: Connection closed: 1039 byte(s) sent to TLS, 15599 byte(s) sent to socket
2019.05.16 17:00:30 LOG5[ui]: Terminated
2019.05.16 17:00:33 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 17:00:33 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 17:00:33 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 17:00:33 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 17:00:33 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 17:00:33 LOG5[ui]: FIPS mode disabled
2019.05.16 17:00:33 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 17:00:33 LOG5[ui]: Configuration successful
2019.05.16 17:00:41 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17891
2019.05.16 17:00:41 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17892
2019.05.16 17:00:41 LOG3[0]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 17:00:41 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:00:41 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53436
2019.05.16 17:00:41 LOG3[1]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 17:00:41 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:00:41 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53440
2019.05.16 17:01:14 LOG5[2]: Service [https] accepted connection from 114.87.93.105:8347
2019.05.16 17:01:16 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:01:16 LOG5[2]: Service [https] connected remote server from 127.0.0.1:53458
2019.05.16 17:01:18 LOG5[3]: Service [https] accepted connection from 114.87.93.105:8349
2019.05.16 17:01:18 LOG5[4]: Service [https] accepted connection from 114.87.93.105:8348
2019.05.16 17:01:18 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:01:18 LOG5[4]: Service [https] connected remote server from 127.0.0.1:53460
2019.05.16 17:01:19 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:01:19 LOG5[3]: Service [https] connected remote server from 127.0.0.1:53462
2019.05.16 17:01:19 LOG5[2]: Connection closed: 1597 byte(s) sent to TLS, 1181 byte(s) sent to socket
2019.05.16 17:01:21 LOG5[3]: Connection closed: 332 byte(s) sent to TLS, 1235 byte(s) sent to socket
2019.05.16 17:01:22 LOG5[4]: Connection closed: 705 byte(s) sent to TLS, 1885 byte(s) sent to socket
2019.05.16 17:02:13 LOG3[1]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.16 17:02:13 LOG5[1]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 17:02:13 LOG3[0]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.16 17:02:13 LOG5[0]: Connection closed: 1039 byte(s) sent to TLS, 15599 byte(s) sent to socket
2019.05.16 17:02:13 LOG5[ui]: Terminated
2019.05.16 17:02:16 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 17:02:16 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 17:02:16 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 17:02:16 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 17:02:16 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 17:02:16 LOG5[ui]: FIPS mode disabled
2019.05.16 17:02:16 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 17:02:16 LOG5[ui]: Configuration successful
2019.05.16 17:02:23 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17916
2019.05.16 17:02:23 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17917
2019.05.16 17:02:23 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:02:23 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53494
2019.05.16 17:02:23 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:02:23 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53496
2019.05.16 17:02:42 LOG5[ui]: Terminated
2019.05.16 17:02:45 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 17:02:45 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 17:02:45 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 17:02:45 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 17:02:45 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 17:02:45 LOG5[ui]: FIPS mode disabled
2019.05.16 17:02:45 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 17:02:45 LOG5[ui]: Configuration successful
2019.05.16 17:03:10 LOG5[ui]: Terminated
2019.05.16 17:03:13 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.16 17:03:13 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.16 17:03:13 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.16 17:03:13 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.16 17:03:13 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.16 17:03:13 LOG5[ui]: FIPS mode disabled
2019.05.16 17:03:13 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.16 17:03:13 LOG5[ui]: Configuration successful
2019.05.16 17:03:33 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17933
2019.05.16 17:03:34 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17934
2019.05.16 17:03:34 LOG3[0]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 17:03:34 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:03:34 LOG5[0]: Service [https] connected remote server from 127.0.0.1:53530
2019.05.16 17:03:34 LOG3[1]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.16 17:03:34 LOG5[1]: s_connect: connected 127.0.0.1:8888
2019.05.16 17:03:34 LOG5[1]: Service [https] connected remote server from 127.0.0.1:53534
2019.05.16 17:04:55 LOG3[1]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.16 17:04:55 LOG5[1]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.16 17:04:55 LOG3[0]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.16 17:04:55 LOG5[0]: Connection closed: 677 byte(s) sent to TLS, 14880 byte(s) sent to socket
